shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -84,7 +84,11 @@ showProfileType
     ;
 
 setVariable
-    : SET variable?
+    : SET (variable? (TO | EQ_) variableValue COMMA_?)*
+    ;
+
+variableValue
+    : (NUMBER_ | identifier | STRING_ | DEFAULT)
```

Comment:
What is the sequence of `NUMBER_ | identifier | STRING_ | DEFAULT`?
How about adjust sequence to `identifier | NUMBER_ | STRING_ | DEFAULT`?

Reply:
done